phoenix-core/src/it/java/org/apache/phoenix/end2end/CreateTableIT.java

```diff
@@ -910,6 +911,41 @@ public void testTableDescriptorPriority() throws SQLException, IOException {
         }
     }
 
+    @Test
+    public void testCreateTableDDLTimestamp() throws Exception {
+        Properties props = new Properties();
+        final String schemaName = generateUniqueName();
+        final String tableName = generateUniqueName();
+        final String dataTableFullName = SchemaUtil.getTableName(schemaName, tableName);
+        String ddl =
+            "CREATE TABLE " + dataTableFullName + " (\n" + "ID1 VARCHAR(15) NOT NULL,\n"
+                + "ID2 VARCHAR(15) NOT NULL,\n" + "CREATED_DATE DATE,\n"
+                + "CREATION_TIME BIGINT,\n" + "LAST_USED DATE,\n"
+                + "CONSTRAINT PK PRIMARY KEY (ID1, ID2)) ";
+        long startTS = EnvironmentEdgeManager.currentTimeMillis();
+        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+            conn.createStatement().execute(ddl);
+            verifyLastDDLTimestamp(schemaName, tableName, dataTableFullName, startTS, conn);
+        }
+    }
+
+    public static long verifyLastDDLTimestamp(String schemaName, String tableName,
+                                              String dataTableFullName, long startTS, Connection conn) throws SQLException {
+        return verifyLastDDLTimestamp("", schemaName, tableName, dataTableFullName, startTS, conn);
+    }
+
+    public static long verifyLastDDLTimestamp(String tenantId, String schemaName, String tableName,
+                                              String dataTableFullName, long startTS, Connection conn) throws SQLException {
```

Comment:
I don't see any usage of first 3 arguments `String tenantId, String schemaName, String tableName`
Maybe you used them in earlier draft then removed them ?